cmd/serve.go

```diff
@@ -550,7 +550,7 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 				if innerErr != nil {
 					log.Ctx(ctx).Fatalf("Error getting scheduler job registrars: %v", innerErr)
 				}
-				go scheduler.StartScheduler(crashTrackerClient.Clone(), schedulerJobRegistrars...)
+				go scheduler.StartScheduler(serveOpts.AdminDBConnectionPool, crashTrackerClient.Clone(), schedulerJobRegistrars...)
```

Comment:
do we need to check AdminDBConnectionPool is initialized?

Reply:
I don't see a reason for this check as we initialized this a few lines above, line 457.